src/browser/transport/xhr.js

```diff
@@ -157,14 +162,12 @@ function _createXMLHTTPObject() {
   var i;
   var numFactories = factories.length;
   for (i = 0; i < numFactories; i++) {
-    /* eslint-disable no-empty */
```

Comment:
🤨

Reply:
Yeah, I presume this was code auto-removing an unused eslint directive, ie. no "no-empty" are ever produced. I'll give it a closer look domani.